projects/packages/plans/src/class-current-plan.php

```diff
@@ -79,6 +79,7 @@ class Current_Plan {
 				'akismet',
 				'payments',
 				'videopress',
+				'field-file', // Forms
```

Comment:
Do we need to specific this both places? Once in wpcom_features and here?

Reply:
I think this takes care of providing the minimum required plan when you are self-hosted or not connected.

Reply:
I don't think we have a personal plan anymore just complete. See https://cloud.jetpack.com/pricing

Reply:
Yeah, I think we'll be better off just pointing at complete for now.